runtime/tr.source/trj9/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -5403,9 +5654,8 @@ J9::Z::TreeEvaluator::VMgenCoreInstanceofEvaluator2(TR::Node * node, TR::CodeGen
    TR::Register                  *objClassReg = NULL;
    TR::Register                  *resultReg = NULL;
    TR::Register                  *castClassReg = NULL;
-   TR::Register                  *helperReturnRegister = NULL;
 
-   // In the evaluator, We need at maximum two scratch registers, so creating a pool of scratch registers with 2 size.
+   // In the evaluator, We need at maximum two scratch registers, so creating a pool of scratch registers with 4 size.
    TR_S390ScratchRegisterManager *srm = cg->generateScratchRegisterManager(2);
```

Comment:
We pass `2` to `generateScratchRegisterManager`. Is this a typo?

Reply:
This is not typo. I calculated number of scratch register I will need live at any point in the evaluator and found that it is maximum two so I only ask srm to allocate two register for me.

Reply:
The comment implies we allocate a pool of scratch registers with size of  `4`. Is the comment incorrect then?

Reply:
Yeah that is incorrect. Thanks for pointing this out. Let me change it.